curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/create-a-form-element.md

```diff
@@ -37,9 +37,8 @@ assert(
 Your `form` should have an `action` attribute which is set to `https://www.freecatphotoapp.com/submit-cat-photo`
```

Comment:
This is kind of nitpicky, but I can imagine coming back as a bug, because of accepting url, which is not exactly the same as in instructions. Would it make sense to add note, that `www.` part of url is optional?

Reply:
So as stated by @ieahleen 

> Now a lot of people that are using old solutions and the videos as guide are failing using not-www links
> Tests need to be made to accept both kind of links in the 18 challenges affected by the PR

I don't really think it should be stated that it's optional only a precaution.